vendor/github.com/tektoncd/pipeline/pkg/apis/config/metrics.go

```diff
@@ -50,7 +50,7 @@ const (
 	// when it isn't specified in configmap
 	DefaultPipelinerunLevel = PipelinerunLevelAtPipeline
 	// PipelinerunLevelAtPipelinerun specify that aggregation will be done at
-	// pipelienrun level
```

Comment:
This file is imported from https://github.com/tektoncd/pipeline - we'll need to make the change upstream and pull it in.

For this PR, you can revert this file.

Reply:
Fixed in pipeline https://github.com/tektoncd/pipeline/pull/5382. Using same title ;)

Reply:
I think we still want to revert the file, right? Anything under vendor will be updated on the next module update.

Reply:
sure thing 🙂

Reply:
updated - would be good if I could get a re-review